exercises/binary_converter.cpp

```diff
@@ -1,7 +1,34 @@
-/*
-  Write a program that given a number as input convert it in binary.
+#include <iostream>
+#include <string>
 
-  Output:
-  Insert first number: 8
-  The binary number is: 1000
-*/
+using namespace std;
+
+string convert(int n);
+
+int main(){
+	int n;
+	cout<<"Insert number to convert:\n";
+	cin>>n;
+	if(n>=0){
+		cout<<"In binary: "<<convert(n)<<"\n";
+	}
+	else{
+		cout<<"Must be positive!\n";
+	}
+}
+
+
+string convert(int n){
+	string s="";
+	do{
+		int x=n%2;
+		if(x==1){
+			s="1"+s;
+		}
+		else{
+			s="0"+s;
+		}
```

Comment:
`+ s` avviene in entrambi i casi quindi potresti scrivere qualcosa come 
```suggestion
		if(x==1){
			s="1";
		}
		else{
			s="0";
		}
		
		s += s;
```

oppure per farla breve
```C++
s = x == 1 ? "1" : "0" + s;
```